recipes/hpx/test/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.18)
+project(HELLO_HPX LANGUAGES CXX)
+
+find_package(HPX REQUIRED)
+
+add_executable(hello_hpx hello_hpx.cpp)
+
+target_link_libraries(hello_hpx
+    HPX::hpx
+    HPX::wrap_main
```

Comment:
There should be no need to explicitly mention `HPX::wrap_main` here. Did you try without it?

Reply:
Without linking to the `wrap_main` target, the header `hpx/hpx_main.hpp` cannot not be found:

```
hello_hpx/hello_hpx.cpp:1:10: fatal error: hpx/hpx_main.hpp: No such file or directory
    1 | #include <hpx/hpx_main.hpp>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
```

See also https://hpx-docs.stellar-group.org/latest/html/quickstart.html?highlight=wrap_main#hello-world

Reply:
Ok, thanks for checking.